### DISCUSSION TITLE
【AI-FastGPT】接入FastGPT 到博客

### DISCUSSION THREAD

**Discussion by tdouguo** [tangly1024/NotionNext#2097] [Ideas]
**为什么提出这个新的特性改动**
文章可以更好的搜索

**描述一下你推荐的解决方案**
https://github.com/labring/FastGPT

**描述一下你考虑过的其它替代解决方案**
直接接官网的 OpenAI的Embedding API，使用Pinecone/Milvus向量库存储向量
https://github.com/huangPengL/ChatGPT-Java-FunAi

**补充说明**
补充与此特性相关的内容

**Comment by dadaoerv** [comment]
请教一下，同样的问题，嵌入代码是：
<html>
<body>


<script>
--
  | window.chatbaseConfig = {
  | chatbotId: "sM5wwryxxxxxxxxxs3BmZ",
  | }
  | </script>
  | <script
  | src="https://www.chatbase.co/embed.min.js"
  | id="sMxxxxxxxxxxxxx3BmZ"
  | defer>
  | </script>


</body>
</html>
这一段代码，我应该加在哪个文件里？
首页效果如下图：
![image](https://github.com/tangly1024/NotionNext/assets/46274596/58702f9d-bbc8-4053-800f-d8911ee86d7a)
感谢大佬

**Reply by wuyuhanzijin** [reply]
同问

**Reply by lifeafter619** [reply]
您只需要将id的值填入您项目的blog.config.js中或使用notionconfig
具体位置在
https://github.com/tangly1024/NotionNext/blob/4535dec0214503c742cbc04c7c7122fcfa12fa55/blog.config.js#L282

**Comment by lifeafter619** [comment]
我想推荐一个ai生成摘要，也希望加入
https://github.com/zhheo/Post-Abstract-AI
